services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/AuthorizationManagerImpl.java

```diff
@@ -280,8 +286,12 @@ public boolean isUserCreatorOrAdmin(UserInfo userInfo, String creator) {
 
 	@Override
 	public AuthorizationStatus canAccessRawFileHandleByCreator(UserInfo userInfo, String fileHandleId, String creator) {

```

Comment:
Add ACCESS_TYPE as a parameter that indicates the purpose for accessing the file handle so that the scope can be mapped against that

Reply:
Done.